src/openapi-generator.ts

```diff
@@ -584,7 +584,7 @@ export class OpenAPIGenerator {
 
     if (
       isZodType(zodSchema, 'ZodEffects') &&
-      zodSchema._def.effect.type === 'refinement'
+      (zodSchema._def.effect.type === 'refinement' || zodSchema._def.effect.type === 'preprocess')
```

Comment:
I would imagine the build checks would fail because of prettier. This should be on two lines

Reply:
Thanks; updated after running `npm run prettier`